example/ios/DemoTests/DemoTests.swift

```diff
@@ -31,7 +31,6 @@ class DemoTests: XCTestCase {
     }
 
     func testExample() {
-        assert(KotlinNativeFramework().hello() == "Hello from the shared module common source")
```

Comment:
As dumb as this testcase is, it does get tested on CI and essentially tests the framework can load etc. Can we replace it with something?